README.md

```diff
@@ -19,6 +19,8 @@ A headless, framework-agnostic and extendable rich text editor, based on [ProseM
 - [tiptap-footnotes](https://github.com/buttondown/tiptap-footnotes) by [@buttondown](https://github.com/buttondown)
 - [tiptap-extension-figma](https://github.com/haydenbleasel/tiptap-extension-figma) by [@haydenbleasel](https://github.com/haydenbleasel)
 - [tiptap-extension-jira](https://github.com/haydenbleasel/tiptap-extension-jira) by [@haydenbleasel](https://github.com/haydenbleasel)
+- [tiptap-slash-command](https://github.com/harshtalks/tiptap-plugins/tree/main/packages/slash-tiptap) by [@harshtalks](https://github.com/harshtalks)
```

Comment:
Please, remove the `by [@harshtalks](https://github.com/harshtalks)` as it does not follow our usual conventions. Sorry for the inconvenience 🙇🏻